### ISSUE TITLE
Heroku build error

### ISSUE THREAD

**Issue by mme** [lfittl/pg_query#32]
When deploying a second time, heroku uses the C source from the build cache which is left in an invalid state from the first build.

After clearing the cache it works again (only once)

```
remote:        
remote:        Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
remote:        
remote:        /tmp/build_9c7939143d416bd33116a081be819d25/vendor/ruby-2.2.2/bin/ruby -r ./siteconf20150826-375-d0iti7.rb extconf.rb
remote:        creating Makefile
remote:        
remote:        make "DESTDIR=" clean
remote:        
remote:        make "DESTDIR="
remote:        compiling pg_query.c
remote:        In file included from /tmp/build_9c7939143d416bd33116a081be819d25/vendor/bundle/ruby/2.2.0/bundler/gems/pg_query-3e40abed0ad5/ext/pg_query/postgres/src/include/postgres.h:48:0,
remote:        from pg_query.h:4,
remote:        from pg_query.c:1:
remote:        /tmp/build_9c7939143d416bd33116a081be819d25/vendor/bundle/ruby/2.2.0/bundler/gems/pg_query-3e40abed0ad5/ext/pg_query/postgres/src/include/utils/elog.h:69:28: fatal error: utils/errcodes.h: No such file or directory
remote:        #include "utils/errcodes.h"
remote:        ^
remote:        compilation terminated.
remote:        make: *** [pg_query.o] Error 1
remote:        
remote:        make failed, exit code 2
remote:        
remote:        Gem files will remain installed in /tmp/build_9c7939143d416bd33116a081be819d25/vendor/bundle/ruby/2.2.0/bundler/gems/pg_query-3e40abed0ad5 for inspection.
remote:        Results logged to /tmp/build_9c7939143d416bd33116a081be819d25/vendor/bundle/ruby/2.2.0/bundler/gems/extensions/x86_64-linux/2.2.0-static/pg_query-3e40abed0ad5/gem_make.out
remote:        An error occurred while installing pg_query (0.6.4), and Bundler cannot
remote:        continue.
remote:        Make sure that `gem install pg_query -v '0.6.4'` succeeds before bundling.
remote:  !
remote:  !     Failed to install gems via Bundler.
remote:  !
remote: 
remote:  !     Push rejected, failed to compile Multipack app
remote: 


```

**Comment by lfittl** [comment]
Interesting - I'll look into cleaning that up.

**Comment by lfittl** [comment]
Fixed by #35
